packages/gatsby-theme-docs/src/overrides/top-menu-banner-area.js

```diff
@@ -0,0 +1,2 @@
+// A React component to be rendered on the right side of the top-menu
+export default () => null;
```

Comment:
According to Nikolaus, the banner should be rendered by default here.

Reply:
So we don't need to override it in the websites at all? 🤔

Reply:
In the smoke tests websites you still need to override the other placeholder (with the github link) to have them visually tested.
The smoke tests websites do not need to override the banner, it's only useful to Nikolaus for the future.

Reply:
Ah right, sorry I ment that we don't need to override the `top-menu-banner-area.js` right?
a456034